dts/bindings/gpio/nxp,pca_series.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 NXP
```

Comment:
need to edit `nxp,pcal6408a.yaml` and `nxp,pcal6416a.yaml` when ready to retire those.